Names/Names/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace Names
+{
+    class Program
+    {
+        static void Main()
+        {
+            var names = new List<string>();
+
+            Console.WriteLine("Обрабатываю данные");
+
+            try
+            {
+                using (StreamReader sr = new StreamReader("names.txt"))
```

Comment:
Не учтен формат данных в прилагаемом файле names.txt. Нужно было разобраться с кодировкой и увидеть, что там все записано в кодировке Win1251.  Чтобы программа обрабатывала наш файл не только в русской Windows, лучше кодировку данных указать явно;
```suggestion
                using (StreamReader sr = new StreamReader("names.txt", Encoding.GetEncoding(1251)))
```